js/main.js

```diff
@@ -17,31 +19,47 @@ $entryForm.addEventListener('submit', function (event) {
   entryFormInfo.title = $entryForm.elements.title.value;
   entryFormInfo.imageUrl = $entryForm.elements.imageUrl.value;
   entryFormInfo.notes = $entryForm.elements.notes.value;
-  entryFormInfo.nextEntryId = data.nextEntryId;
-  data.nextEntryId += 1;
 
-  data.entries.unshift(entryFormInfo);
+  if (editTime) {
 
-  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+    entryFormInfo.nextEntryId = data.editing.nextEntryId;
+    data.editing = entryFormInfo;
+
+    var $editedEntry = document.querySelectorAll('.entry')[data.entries.length - data.editing.nextEntryId];
```

Comment:
You should not depend on an `id` to be the same as its index like this. For instance, what if eventually the item with the id `4` gets deleted? Now this won't work. Instead, you should loop over your list of entries and find the entry with the matching id.